src/layouts/shortcodes/first_azure_nodepools_version.html

```diff
@@ -0,0 +1 @@
+12.1.0
```

Comment:
This is so we could use `{{% first_azure_nodepools_version %}}` to get first Azure NP release.